Views/Home/Index.cshtml

```diff
@@ -1,5 +1,24 @@
 <div class='main'>
     <img class='phone-img' src='~/images/phone.png' alt='Phone'/>
-    <div class='separator'></div>
+    <div class='separator-y'></div>
     <img class='logo-img' src='~/images/tcg_logo.png' alt='Logo'/>
 </div>
+
+<div class='separator-x'></div>
+<div class='features'>
+    <div class='feature'>
+        <img class='img' src='~/images/ic_list.png' alt='ic_list'/>
+        <p class='title'>Keep track of your money</p>
```

Comment:
this is a title, you should use a headline tag for this instead of using a paragraph tag.